src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -232,14 +235,12 @@ public void setFormats(List<String> formats) {
         this.formats = formats;
     }
 
-    public void setAuthorDisplayName(Author author, String displayName) {
+    public void addAuthorDisplayName(Author author, String displayName) {
         authorDisplayNameMap.put(author, displayName);
```

Comment:
there should only be a single `displayName` for any `author` right?
In this case, the `map.put()` method will replace the `displayName` for the `author`, so this should be a `set` instead of `add`.